Update and rename Readme.md to onboarding.md (#102)
mattermost/mattermost-handbook: contributors/onboarding/onboarding.md

---
description: >-
  1% complete. Onboarding information and checklists for general staff and
  departments
---

# Onboarding

> Welcome to Mattermost Onboarding! The goal of the Onboarding Program is to smoothly onboard and enable new colleagues into a safe and compliant working environment with a strong sense of belonging within their first 90 days at Mattermost. 

Reviewer: ```suggestion
Welcome to Mattermost Onboarding! The goal of the Onboarding program is to welcome and enable new colleagues into a safe and compliant working environment with a strong sense of belonging within their first 90 days at Mattermost. 
```

Author: Looks great!

Onboarding is an all-company partnership between a new colleague and Mattermost. Specifically, a new colleague’s manager, functional and working teams, onboarding [peers](https://docs.google.com/document/d/1DQULEvgOhGeEGLTzdOlSvo858vo94zmnMN8sygi-lHk/edit?ts=5e21f202), [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT. 

### Post-Onboarding Readiness 

> Post Onboarding Readiness is an internal measure based on feedback throughout a new team member’s first 90 days at Mattermost. There are multiple milestones throughout the onboarding process where feedback will be solicited. This feedback intends to assess how willing and enabled a new colleague is to bring their best self to Mattermost as it relates to their role expectations, team and company focuses and the Mattermost Leadership Principles. 

Reviewer: ```suggestion
Post-Onboarding Readiness is an internal measure based on feedback throughout a new team member’s first 90 days at Mattermost. There are multiple milestones throughout the onboarding process where feedback is solicited. This feedback assesses how willing and enabled a new colleague is to bring their best self to Mattermost as it relates to their role expectations, team and company focuses, and the Mattermost Leadership Principles. 
```

> Feedback will be requested throughout the 90-day Onboarding These consistent check-ins help new colleagues, HR and managers iterate with impact quickly. For example, one of the [V2MOM - Destination Workplace Measures](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f2ea8cda4_8_1852) is to have fewer than 5% questions repeat month-over-month. 

Reviewer: ```suggestion
Feedback is requested from the new colleague's peers throughout the 90-day Onboarding. These consistent check-ins help new colleagues, HR, and managers iterate with impact quickly. For example, one of the [V2MOM - Destination Workplace Measures](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f2ea8cda4_8_1852) is to have fewer than 5% questions repeat month-over-month. 
```

> #### Onboarding Feedback to assess post-onboarding readiness 

Reviewer: ```suggestion
#### Onboarding Feedback to Assess Post-Onboarding Readiness 
```

**Onboarding Surveys**
TBD

**New Staff Feedback Session with CEO and HR**

> Similar to [CEO listening tours](https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours) new colleagues will be asked to share [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) about anything Mattermost and Onboarding. 

Reviewer: ```suggestion
Similar to [CEO listening tours](https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours) in this session, new colleagues are asked to share [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) about anything related to Mattermost and the Onboarding Program. 
```

**New Colleague Feedback**

> The 90-Day Feedback provides a [360](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared) [shoulder check](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) for new team members, managers and [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT to understand and clarify potential blindspots as well as catch potential [brown m&ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) early on. 

Reviewer: ```suggestion
The 90-Day Feedback provides a [360](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared) [shoulder check](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) for new team members, managers, [HR](https://handbook.mattermost.com/operations/workplace/people#team), and MLT to understand and clarify potential blindspots as well as catch potential [brown M&Ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) early on. 
```

> This is not a formal performance review. The intent is to align expectations and improve performance early on. Peer feedback is not anonymous. Managers may request additional context from the panel to help increase the new colleague's performance and post-onboarding readiness. Managers should evaluate the peer feedback against their own expectations of the new colelague's role. In some cases, expectations may need to be re-set with peers. Uncovering blindspots early on helps avoid future confusion or misfocused efforts. 

Reviewer: ```suggestion
This is not a formal performance review. The intent is to align expectations and improve performance early on. Peer feedback is not anonymous. Managers may request additional context from the panel to help increase the new colleague's performance and Post-Onboarding Readiness. Managers should evaluate the peer feedback against their own expectations of the new colleague's role. In some cases, expectations may need to be reset with peers. Uncovering blindspots early on helps avoid future confusion or misaligned efforts. 
```

**New Colleague 90-Day Feedback Process**

 * **Managers**

>     * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employmnet. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 

Reviewer: ```suggestion
    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employment. This kicks off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
```
Missing a link for the new colleague peer panel form.

>     * Managers will recieve a unique link with peer panel results once the survey is sent. 

Reviewer: ```suggestion
    * Managers receive a unique link with peer panel results after the survey is sent. 
```

>     * People Ops will share the new colleague's self evaluation once completed. 

Reviewer: ```suggestion
    * People Ops will share the new colleague's self-assessment once completed. 
```

Author: Thank you for catching this!

>     * People Ops will share the new colleague's self evaluation once completed. 

Reviewer: ```suggestion
    * People Ops share the new colleague's self-assessment once completed. 
```

 * **New Colleagues**

>     * Complete the [90-Day New Colleague Self Assessment](https://forms.gle/vyZeGCfG57BoGuhB9). Notifications are sent on day 75. 

Reviewer: ```suggestion
    * Complete the [90-Day New Colleague Self-Assessment](https://forms.gle/vyZeGCfG57BoGuhB9). A notification is sent on day 75 with a reminder to complete the assessment. 
```

---
description: >-
  1% complete. Onboarding information and checklists for general staff and
  departments
---

# Onboarding

Welcome to Mattermost Onboarding! The goal of the Onboarding program is to welcome and enable new colleagues into a safe and compliant working environment with a strong sense of belonging within their first 90 days at Mattermost. 

> Onboarding is an all-company partnership between a new colleague and Mattermost. Specifically, a new colleague’s manager, functional and working teams, onboarding [peers](https://docs.google.com/document/d/1DQULEvgOhGeEGLTzdOlSvo858vo94zmnMN8sygi-lHk/edit?ts=5e21f202), [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT. 

Reviewer 2: Onboarding is an all-company partnership between a new colleague and Mattermost. Specifically, a new colleague’s manager, functional and working teams, onboarding [peers](https://docs.google.com/document/d/1DQULEvgOhGeEGLTzdOlSvo858vo94zmnMN8sygi-lHk/edit?ts=5e21f202), [HR](https://handbook.mattermost.com/operations/workplace/people#team) and [MLT](https://handbook.mattermost.com/operations/workplace/groups).

Author: Thanks @hannaparks!
